module-1/lab-api-scavenger-game/your-code/challenge-2.ipynb

```diff
@@ -0,0 +1,1520 @@
+{
```

Comment:
As in challenge 1, try to normalize JSONs. Your code has worked because of the field you have chosen to make the final query. Not normalizing JSON could throw undesired results because you loose nested dictionaries. 
Remember that a JSON is like a python dictionary!!